### ISSUE TITLE
Oxygen tank bombs not working

### ISSUE THREAD

**Issue by Richie9999** [tgstation/tgstation#70231]
Reporting client version: 514.1588

## Round ID:
191505

## Reproduction:

Create oxygen tank bomb. Send signal to detonate. Bomb goes *beep**beep**beep* then fizzles and disappears. No explosion logged in admin logs

**Comment by Jackal-boop** [comment]
Did you bore a pressure hole into it?

**Comment by Richie9999** [comment]
Yup. To be perfectly fair, bomb making isn't part of my expertise. Made this bug report based on an ahelp I picked up that I then tried out myself prior to submitting the report. Player who first ahelped it disconnected before I could suggest to them that they make a bug report.

**Comment by calliophis-maculiceps** [comment]
pressure temperature and gas mixture please

**Comment by Sealed101** [comment]
no gas mix to back this up equals can't reliably repro
